src/share/constants.hpp

```diff
@@ -23,7 +23,7 @@ class constants final {
     return "/Library/Application Support/org.pqrs/tmp/karabiner_event_dispatcher_receiver";
   }
 
-  static const char* get_home_dot_karabiner_directory(void) {
+  static const char* get_configuration_directory(void) {
```

Comment:
Note that this should first check (and use, if set) `$XDG_CONFIG_HOME` for full compatibility. (I don't personally care for that, but some users might.)

(See https://github.com/tekezo/Karabiner-Elements/issues/112#issuecomment-249102921)